Update/some_004.txt

```diff
@@ -2641,7 +2641,7 @@ void main()
 			NULL, "\"B-but about the other day!", Line_WaitForInput);
 	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100378", 256, TRUE);
 	OutputLine(NULL, "　こ、この前のセンターで会ったのはその、アルバイトじゃなくって、",
-			NULL, " W-when I met you at the center, it wasn't a part-time job,", Line_WaitForInput);
+			NULL, " W-when I met you at the care and welfare center, it wasn't a part-time job,", Line_WaitForInput);
```

Comment:
Expanding to "care and welfare center" might make the dialogue seem a bit less naturalistic here. They've only met at one center, after all, so the context is pretty clear. (And the original, so far as I can read, also limits itself to something along the lines of "Before, when I met you at the center...")